packages/block-library/src/audio/edit.js

```diff
@@ -115,7 +121,7 @@ class AudioEdit extends Component {
 		if ( editing ) {
 			return (
 				<MediaPlaceholder
-					icon="media-audio"
+					icon=<BlockIcon icon={ icon } />
```

Comment:
Can it follow other definitions of props?:
```suggestion
					icon={ <BlockIcon icon={ icon } /> }
```